### PR TITLE
Update Scala, plugins and dependencies versions

### PR DESCRIPTION
**Pull request by bio-aeon**
- Scala
- sbt
- plugins
- dependencies

### CODE REVIEW

**Comment by bio-aeon** [comment]
@cb372 @LukaJCB Could I please get approval for PR workflows to unblock?

**Review by LukaJCB — APPROVED** [review]

**Comment by LukaJCB** [comment]
Sorry for the delay!

**Review by cb372 — APPROVED** [review]
Thank you!

**Pull request merged**
